### DISCUSSION TITLE
teeAction() applied to a specific dnsdist server

### DISCUSSION THREAD

**Discussion by deanfr** [PowerDNS/pdns#14848] [Q&A]
Hi,

I use a pool backend containing a main server + a backup server.
I would like to apply teeAction() to all requests sent to the backup server

I would prefer to avoid defining a backup pool server and apply the teeAction() to requests matching this pool server

**Comment by rgacogne** [comment]
I'm afraid this is not possible, the selection of a backend inside a pool is done after the rules have been processed.

**Comment by deanfr** [comment]
ok, thank's Remi.
